lib/serve/rest-api/src/api/routes.py

```diff
@@ -30,6 +31,9 @@
 router.include_router(models.router, prefix="/v1", tags=["models"], dependencies=[Depends(security)])
 router.include_router(embeddings.router, prefix="/v1", tags=["embeddings"], dependencies=[Depends(security)])
 router.include_router(generation.router, prefix="/v1", tags=["generation"], dependencies=[Depends(security)])
+router.include_router(
+    litellm_passthrough.router, prefix="/v2/serve", tags=["litellm_passthrough"], dependencies=[Depends(security)]
```

Comment:
I'm assuming this is `/v2/serve` because we're going to move the API GW stuff into `/v2/` as well?

Reply:
Yes, that is correct. I'm also welcome to changing the name, but I wanted to avoid the situation of calling it "model" and having a url like the following when using it directly for the OpenAI list models call:

```
curl -X GET https://mydomain/v2/model/models
```